QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
```

Comment:
I think this stemmed from me thinking that we wouldn't create a new instance of NotifyAllListeners(), so it could be static, which meant _listeners had to be static as well.